src/main/java/net/openhft/chronicle/network/AlwaysStartOnPrimaryConnectionStrategy.java

```diff
@@ -51,6 +51,9 @@
     private int socketConnectionTimeoutMs = Integer.getInteger("connectionStrategy.socketConnectionTimeoutMs", 1);
     private long pauseMillisBeforeReconnect = Integer.getInteger("connectionStrategy.pauseMillisBeforeReconnect", 500);
     private ClientConnectionMonitor clientConnectionMonitor = new VanillaClientConnectionMonitor();
+    private long minTimeSec = Integer.getInteger("connectionStrategy.pause.min.secs", 5);
```

Comment:
I think you only need to change this file?

Reply:
Please always use Jvm.getInteger() because it allows code user to set those properties in advance in controlled fashion.

Reply:
Thanks, already did, see the last changes.